packages/backend-tasks/src/tasks/TaskWorker.ts

```diff
@@ -63,7 +59,7 @@ export class TaskWorker {
             break;
           }
 
-          await sleep(WORK_CHECK_FREQUENCY, options?.signal);
+          await sleep(this.workCheckFrequency, options?.signal);
```

Comment:
You forgot a changeset for this package right?

Reply:
Yep!